### PR TITLE
Add Justine Geffen, Eric Sadur and Adam Clarkson to core committers + R&D staff list

### PR DESCRIPTION
**Pull request by jasonblais**
I'm not sure who usually reviews, but I see @lieut-data you've made some changes recently so requesting your review. Amy's for editor review.

----

FYI @justinegeffen @adamjclarkson @etoaster 

Adam/Eric: For R&D list, I added you under "Other" while we figure out how the new product and engineer structure work together.

### CODE REVIEW

**Comment by amyblais** [comment]
@jasonblais Not related to this PR but if you have time could you remove ``Amy Blais`` from the list of Product Managers? It's very confusing.

**Comment by jasonblais** [comment]
@amyblais submitted: https://github.com/mattermost/mattermost-developer-documentation/pull/451

File: site/content/internal/rd-teams.md [hunk 1]
```diff
@@ -103,3 +103,6 @@ weight: 110
 * Jelena Gilliam - QA
 * Ogi Marusic - QA
 * Steve Mudie - QA
+* Justine Geffen - Technical Writer
+* Adam Clarkson - PM
```
Comment (lieut-data) [inline site/content/internal/rd-teams.md]: Does it make sense to list the PM with their associated team(s)?

Comment (jasonblais) [reply site/content/internal/rd-teams.md]: They're not yet mapped to a team, from PR description: `Adam/Eric: For R&D list, I added you under "Other" while we figure out how the new product and engineer structure work together.`

Comment (lieut-data) [reply site/content/internal/rd-teams.md]: Hah, teaches me to read the whole description next time -- thanks for the gentle reminder :)

**Pull request merged**